keyboards/wilba_tech/wt65_a/readme.md

```diff
@@ -1,8 +1,8 @@
 # WILBA.TECH WT65-A
 
-![WILBA.TECH WT65-A](https://wilba.tech)
+![WILBA.TECH WT65-A](https://images.squarespace-cdn.com/content/5b267e429772ae4372e3b65f/1556352502542-AKHBF9R6CP5HVHOZYJHM/i69_g.png?content-type=image%2Fpng)
```

Comment:
I think you meant to put this on `keyboards/wilba_tech/wt69_a/readme.md` Line 3.

Though this should be an image URL too.

```suggestion
![WILBA.TECH WT65-A](https://wilba.tech/images/wt65a.jpg)
```

Reply:
whoops